Dockerfile

```diff
@@ -1,16 +1,17 @@
 FROM node:alpine
 
-ARG APP_PORT
+ARG SERVER_PORT
 
 WORKDIR /app
 COPY . ./
 
 ENV PATH ./node_modules/.bin:$PATH
+ENV NODE_ENV "development"
 
 USER node
 
 RUN mkdir -p /app/node_modules
 
-EXPOSE $APP_PORT
+EXPOSE $SERVER_PORT
```

Comment:
@IAmRGroot Why is this necessary by the way? Can't we just remove this, expose some default port (like 3000), and just allow the user to change the local port that's bound to this port in `docker-compose.yaml` like so:

```yaml
environment:
  SERVER_PORT: ${SERIVER_PORT:-3000}:3000
```

Reply:
I think we can change it to 3000 default. It's probably there because of some earlier traefik struggles